test/snapshots/lib-buildMockGroupFieldMap.test.ts.md

```diff
@@ -31,6 +31,7 @@ Generated by [AVA](https://avajs.dev).
             placeholder: 'Commodo elit at',
             select: 'document',
             tags: undefined,
+            text: undefined,
```

Comment:
❓ #question: Could you verify that the API returns `undefined` if `text` is not enabled?

Similarly, is it possible for the API to return `null` if the link text is filled in and then cleared, similar to a key text field? Do we need to add support for that use case in `@prismicio/mock`?

Reply:
@angeloashmore Verified that it returns `undefined` when `text` is not enabled. It's not possible for the API to return `null`. This is the intentional behavior that we decided on.

Reply:
Perfect! 👌